### DISCUSSION TITLE
It seems that LaTeX is not working.

### DISCUSSION THREAD

**Discussion by littledot2020** [open-webui/open-webui#2301] [General]
In the conversation, I asked chatbot to output complex numerical formulas, but they are not displayed in the normal formula format. Instead, the code formula effect is shown. I took a few screenshots.

I didn't know how to classify this issue, so I didn't select a problem template.

![image](https://github.com/open-webui/open-webui/assets/43518084/ae67351e-5ea0-4850-9321-1d3675631fb3)

![image](https://github.com/open-webui/open-webui/assets/43518084/16c24d3e-3d4c-4614-a1c2-275180e29277)

**Comment by Hunter6324** [comment]
Indeed, I 
![Uploading 屏幕截图 2024-05-16 153927.png…]()
found out as well

**Comment by chrisoutwright** [comment]
It is also not rendering  \(  \)  , inlines, single line \[ \] works though

![SmartSelect_20240715_004803_Chrome](https://github.com/user-attachments/assets/e2964140-929b-43b5-b18d-9e796abe0b29)

**Reply by chrisoutwright** [reply]
original text: 
```
You're right to ask! If there's no bias, meaning both dice behave identically (both are fair), then the KL divergence will be zero.

Let's see why:

* **Identical Distributions**: When \( p = q \) (meaning the probability distribution of rolling each face is the same for both dice), then \( \frac{p(x)}{q(x)} = 1 \).
* **Logarithm of 1**:  The logarithm of 1 (log 1) is always zero.

Therefore, in the formula:

\[ D_{KL}(p||q) = \sum_x p(x)\log\frac{p(x)}{q(x)} \]

Each term \( p(x)\log\frac{p(x)}{q(x)} \) will be zero since \( \frac{p(x)}{q(x)} = 1 \). The sum of zeros is still zero.


**Conclusion:** When the two distributions are identical (like two fair dice), the KL divergence between them is zero, indicating no information gain from switching between them.
```

**Reply by chrisoutwright** [reply]
It is rather uncomfortable reading like this, I which there would be some rule-based processor or small ML correcting that on the fly if there is anything wrong for formatting latex/equations but I do not see an issue with the inlines.
